backend/app/services/adapters/skill_kinds.py

```
@@ -370,6 +370,92 @@ def create_skill(
+        If a skill with the same name already exists in the target namespace
+        for this user, returns the existing skill without creating a duplicate.
+
+        Returns:
+            Dict with keys:
+              - original_id: source skill Kind.id
+              - target_id: destination skill Kind.id (new or existing)
+              - was_copied: True if a new skill was created, False if existing reused
+        """
+        source = (
+            db.query(Kind)
+            .filter(Kind.id == skill_id, Kind.kind == "Skill", Kind.is_active == True)
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)